backend/package.json

```diff
@@ -1,38 +1,43 @@
 {
-  "name": "indaba",
-  "description": "",
-  "version": "0.2.0",
-  "private": true,
-  "dependencies": {
-    "express": "4.10.2",
-    "body-parser": "1.9.2",
-    "multer": "0.1.3",
-    "aws-sdk": "2.0.27",
-    "pg": "3.6.3",
-    "sql": "0.48.0",
-    "co": "4.0.0 ",
-    "mz": "1.1.0",
-    "thunkify":"2.1.2",
-    "underscore": "1.7.0",
-    "async": "0.9.0",
-    "chai": "1.9.1",
-    "faker": "1.0.0",
-    "mocha": "1.20.1",
-    "node-uuid": "1.4.1",
-    "passport": "0.2.1",
-    "passport-http": "0.2.2",
-    "supertest": "0.13.0",
-    "validator": "3.16.0",
-    "verror": "1.4.0",
-    "winston": "0.8.3",
-    "diff-json": "*",
-    "ejs-locals": "1.0.2",
-    "nodemailer": "1.3.0",
-    "nodemailer-smtp-transport": "0.1.13",
-    "moment" : "*",
-    "accept-language" : "2.0.16"
-  },
-  "scripts": {
-    "start": "node --harmony app.js"
-  }
-}
+    "name": "greyscale-backend",
```

Comment:
I only did this because the client `package.json` uses `greyscale-client`. I'm fine with either; you're correct that we need to be consistent.